tsconfig.base.json

```diff
@@ -16,85 +16,85 @@
     "baseUrl": ".",
     "rootDir": ".",
     "paths": {
-      "@nrwl/angular": ["./packages/angular"],
-      "@nrwl/angular/*": ["./packages/angular/*"],
-      "@nrwl/cli": ["./packages/cli"],
-      "@nrwl/cli/*": ["./packages/cli/*"],
-      "@nrwl/cypress": ["./packages/cypress"],
-      "@nrwl/cypress/*": ["./packages/cypress/*"],
-      "@nrwl/detox": ["./packages/detox"],
-      "@nrwl/devkit": ["./packages/devkit"],
-      "@nrwl/devkit/ngcli-adapter": ["./packages/devkit/ngcli-adapter"],
-      "@nrwl/devkit/testing": ["./packages/devkit/testing"],
-      "@nrwl/e2e/cli": ["./e2e/cli"],
-      "@nrwl/e2e/utils": ["./e2e/utils"],
-      "@nrwl/eslint-plugin-nx": ["./packages/eslint-plugin-nx/src"],
-      "@nrwl/express": ["./packages/express"],
-      "@nrwl/jest": ["./packages/jest"],
-      "@nrwl/jest/*": ["./packages/jest/*"],
-      "@nrwl/js": ["./packages/js/src"],
-      "@nrwl/js/*": ["./packages/js/*"],
-      "@nrwl/linter": ["./packages/linter"],
-      "@nrwl/nest": ["./packages/nest"],
-      "@nrwl/next": ["./packages/next"],
-      "@nrwl/node": ["./packages/node"],
-      "@nrwl/node/*": ["./packages/node/*"],
+      "@nrwl/angular": ["packages/angular"],
+      "@nrwl/angular/*": ["packages/angular/*"],
+      "@nrwl/cli": ["packages/cli"],
+      "@nrwl/cli/*": ["packages/cli/*"],
```

Comment:
`swc-node` doesn't resolve these paths properly if they start with './', but it shouldn't be necessary and when generating new workspaces we don't use the `./`, so we can swap these out safely